validphys2/src/validphys/mc_gen.py

```diff
@@ -91,8 +93,9 @@ def art_data_moments(art_rep_generation, color="green"):
     _, _, normart_replicas, _ = art_rep_generation
 
     artrep_array = np.asarray(normart_replicas)
-
-    fig, axes = plt.subplots(nrows=3, figsize=(10, 12))
+    
+    fig = Figure(figsize=(10, 12))
+    axes = [fig.add_subplot(3, 1, i+1) for i in range(3)]
```

Comment:
Why does this file uses `Figure` directly instead of relying in `plotutils`?

(I remember previous pain when dealing with this file so I guess it does something strange that plotutils didn't account for? Should it?)

Reply:
Yes, the main reason is that list comprehension. So the easiest thing to do was to use Figure